Review.md

```diff
@@ -3,10 +3,28 @@
 You can exercise your Google-Fu for this and any other _Sprint Challenge_ in the future.
 
 1.  Explain the difference between `RDBMS` and `SQL`.
+
+RDBMS is a MANAGEMENT SYSTEM - it's the database software itself, so it handles storage, querying, and updating the data in a db. SQL is a LANGUAGE that is used to interact with the system. So for example, SQLite is the RDBMS and SQL is the language you'd use to interact with it.
+
 1.  Why do tables need a `primary key`?
+
+Tables need primary keys so that each item in the table can have a unique identifier. The identifier shouldn't have any relation to the rest of the data in a table. The primary key should never change.
```

Comment:
>The identifier shouldn't have any relation to the rest of the data in a table.

I'm not sure if this is exactly true. I'm asking people smarter than me for clarification. Database administration is serious business. :joy: